docs/devel/development.md

```diff
@@ -116,8 +116,11 @@ git remote set-url --push upstream no_push
 
 ### Committing changes to your fork
 
-Before committing any changes, please link/copy these pre-commit hooks into your .git
-directory. This will keep you from accidentally committing non-gofmt'd Go code.
+Before committing any changes, please link/copy the pre-commit hook
+into your .git directory. This will keep you from accidentally
+committing non-gofmt'd Go code. In addition this hook will do a build.
```

Comment:
Flow with the added sentence is kind of funky, but I want people to expect that this commit hook will take a while. I was surprised the first time I tried to run this, as the description was "run gofmt", so I expected it to run instantly, not hang for a while.